packages/filecoin-actor-utils/data/actor-descriptors.json

```diff
@@ -0,0 +1,3023 @@
+{
```

Comment:
@navFooh `wallaby` network here also has an `evm` built-in actor - which makes me think of a bigger thing - how do we handle cases where the actor-descriptor is different on a per network basis? I.e. what if the `multisig` actor changes the interfaces of one of its method, and that is live on wallaby/calibration, but not yet on mainnet?

Reply:
Yeah that is tricky. I'm guessing the actor descriptor are prone to change for the same network as well. So a message from a year ago might need different interpretation for the params and return values?

If there are different Go specs for the different networks, we might be able to deal with this programmatically

Reply:
> So a message from a year ago might need different interpretation for the params and return values?

Good question... Let me do some asking around on this. I'm pretty sure old messages still have to be able to be "validated" by miners in order for the network to remain in tact, which makes me think that this must be handled somehow at the protocol level.